src/index.js

```diff
@@ -14,15 +14,14 @@ let initModule = false;
  * @param  {string} [config.LOGS_TOKEN] Token used by internal logger to send logs to
  * @return {Object}        The configured providers
  */
-module.exports = function(config) {
-  //if no param is provided, we want to return the cached modules
+module.exports = (config) => {
+  // if no param is provided, we want to return the cached modules
   if (!config) {
```

Comment:
I think ti is weird usage , although maybe na edge case but should be treated the same as not supplying any param because as a caller you have not specified what you need and maybe you just want the previously inited  version like in the case of require('boot-service')().